api/models/post.js

```diff
@@ -14,9 +14,13 @@ const PostSchema = new mongoose.Schema(
       required: true,
     },
     message: { type: String, required: true },
-    likes: [{type: mongoose.Types.ObjectId, ref:'User'}]
```

Comment:
I see you've slightly changed this, was the previous implementation causing any issues?